### ISSUE TITLE
Drupal 8 Install using Minimal profile redirects to install.php after installation.

### ISSUE THREAD

**Issue by ttrowell** [pantheon-systems/drops-8#96]
Expected:
After site installation with the Minimal profile, a success message is given and the link to Visit the new site is presented.  (standard core process)

Actual:
Requests to to the site after installation are routed back to the install.php page where the 'Drupal is already installed' messaging appears.

Issue in action:
https://youtu.be/lHAZ6U9d2cw

**Comment by alphex** [comment]
+1.
This is happening to me as well.

Create pantheon D8 instance.
Keep it in SFTP mode for install process.
Complete install process, after filling out the configuration screen it sends me to the "Drupal already installed" warning.

When I click the link for "View your existing site." it sends me back to the "Drupal already installed" warning page again.

![2015-10-14 16-06-14](https://cloud.githubusercontent.com/assets/597641/10496191/80a07d52-728d-11e5-9c96-0424030025bd.jpg)

**Comment by greg-1-anderson** [comment]
This happens because pantheon.settings.php uses an inaccurate test to determine whether or not the site has been installed before.

Workaround:

Remove the block of code in settings.pantheon.php that calls 'install_goto("core/install.php");' and instead, manually enter core/install.php in the URL bar to install your site the first time.

The correct long-term solution would be to [fix the redirect logic in Drupal core to work on Pantheon](https://github.com/pantheon-systems/drops-8/issues/3#issuecomment-130776789).

**Comment by alphex** [comment]
If it helps anyone else... This is the block of code I commented out to get through the Install successfully.

![2015-10-14 17-23-04](https://cloud.githubusercontent.com/assets/597641/10498193/54435594-7298-11e5-9314-1f62a27f70e1.jpg)

Thanks Greg.

**Comment by greg-1-anderson** [comment]
One solution to this problem would be #3. Another would be to remove the redirect to core/install.php altogether, and make the "Visit Site" button go there if the site is not installed yet.

**Comment by TommyWillB** [comment]
I tried commenting out those lines, but now my entire site is Access Denied, including the login:

settings.pantheon.php:
```php
/** if (
*   isset($_ENV['PANTHEON_ENVIRONMENT']) &&
*   !$is_installer_url &&
*   (!is_dir(__DIR__ . '/files/styles')) &&
*   (empty($GLOBALS['install_state'])) &&
*   (php_sapi_name() != "cli")
* ) {
*   include_once __DIR__ . '/../../core/includes/install.core.inc';
*   include_once __DIR__ . '/../../core/includes/install.inc';
*   install_goto('core/install.php');
* }
*/
```

**Comment by alphex** [comment]
Did you do it before you ran any of the install.php process?

When I got it to work, I did the site spin up, and before visiting the DEV
site, I did my git clone and altered the file, before running any of the
install.

(maybe pantheon has fixed this? since they're officially supporting d8 now?)
ᐧ


] duran goodyear
] alphex information solutions, llc
] w: http://alphex.com // v: 267.225.7439 // e: duran@alphex.com

**Comment by greg-1-anderson** [comment]
If you remove the above-mentioned lines from settings.pantheon.php, and then visit your site, you should get the following error:
```
The website encountered an unexpected error. Please try again later.
Drupal\Core\Database\DatabaseExceptionWrapper: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'pantheon.sessions' doesn't exist: SELECT session FROM {sessions} WHERE sid = :sid LIMIT 0, 1; Array ( [:sid] => ZunbF_4Ggq-vz1xiatsXNLYJBDA3BjA-h8P5PaATYdE ) in Drupal\Core\Session\SessionHandler->read() (line 66 of core/lib/Drupal/Core/Session/SessionHandler.php).
```
If you get any other result, go back to your dashboard and wipe the site, and try again.

To actually install, go directly to /core/install.php. You should then be able to install the Minimal profile. Just tried this with #110, and it works great.

Again, the real solution will be one of the variants suggested in #3.

**Comment by greg-1-anderson** [comment]
This is now fixed on Drupal 8.0.0 as of today.
